Sources/SteamPress/Controllers/BlogController.swift

```diff
@@ -30,12 +30,20 @@ struct BlogController {
         drop.group(pathCreator.blogPath ?? "") { index in
             index.get(handler: indexHandler)
             index.get(blogPostsPath, String.self, handler: blogPostHandler)
-            index.get(tagsPath, String.self, handler: tagViewHandler)
-            index.get(authorsPath, String.self, handler: authorViewHandler)
             index.get(apiPath, tagsPath, handler: tagApiHandler)
             index.get(blogPostsPath, handler: blogPostIndexRedirectHandler)
-            index.get(tagsPath, handler: allTagsViewHandler)
-            index.get(authorsPath, handler: allAuthorsViewHandler)
+            
+            let disabledPaths = config.disabledPaths
```

Comment:
Not sure I like this as it 'stringly' types the settings for this. As there are only two things that can be disabled I would prefer a `Bool` to be injected in for each of the settings. Gives you some compiler safety and keeps the responsibility of the configuration in the Provider to stop it leaking into the controller. Could you change it to be `enableTagsPages` and `enableAuthorsPages` being passed in and defaulting to `true`?